python/hail/api2/table.py

```diff
@@ -766,7 +767,7 @@ def select(self, *exprs, **named_exprs):
         for k, e in named_exprs.items():
             all_exprs.append(e)
             analyze('Table.select', e, self._row_indices)
-            strs.append('`{}` = {}'.format(k, to_expr(e)._ast.to_hql()))
+            strs.append('{} = {}'.format(escape_id(k), to_expr(e)._ast.to_hql()))
 
```

Comment:
Do you need to escape the key for `group_by`, `aggregate`, and `explode`?

Reply:
group by yes, and it's escaped. Aggregate and explode, no. They don't build named exprs